### ISSUE TITLE
Easier instance switching

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#11]
Right now you have to go into the settings to switch instances, but it'd be nice if you could do it from the home page.

**Comment by nolanlawson** [comment]
My current thinking:

1. Tap the account picture on the home timeline
2. A switcher modal pops up
3. Tap the picture again to go to your account profile, or tap another picture to go to that instance's timeline.

**Comment by selfagency** [comment]
i think it makes more sense to put it in the menu bar, along with a logout link

**Comment by nolanlawson** [comment]
The menu bar is already stuffed a bit full; not sure how we could do that without moving the search button or something else out (and we may have to do that for pinning two timelines instead). Although maybe it could be placed there in desktop/large-screen only.

**Comment by mbajur** [comment]
Pretty logical way of dealing with that (based on other services) would be to create an avatar link on the right side of navbar. It would have a dropdown with: 

- My profile
- Settings (and remove settings from the main navbar links)
- Switch account
- Logout

**Comment by shellkr** [comment]
Yes, that could work... but it the avatar would have to follow scrolling down like the compose button.

Another solution would be to add a '+' symbol in the home tab and clicking on it would let you chose account.

**Comment by nolanlawson** [comment]
#945 gets us a bit closer, but I'd like to add buttons and/or hotkeys right on the home timeline (or any page potentially). Something like the way Slack does it (with a sidebar on the left) seems reasonable for desktop at least.

**Comment by Eoin-ONeill-Yokai** [comment]
@nolanlawson Any way we could just have a shortcut for instance for switching through all active instances? Something like a tab / alt-tab behavior but for instances?

**Comment by bluebirch** [comment]
I can't seem to find a way to switch instances with keyboard only. Is there such a way?

**Comment by MarcoZehe** [comment]
The only way currently is with a lot of tabbing to Settings, To Instances, and then to the radio button of the instance you want to switch to. Very cumbersome indeed.

**Comment by jdaviescoates** [comment]
Yeah, switching accounts/ instances could be easier, but the main reason I came here was just to say that it's currently too hard to find the logout button (it's hidden in Settings -> Instances -> click on the instance name -> scroll to the bottom -> Logout button).
